src/index.html

```diff
@@ -9,6 +9,70 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Moyo header</h1>
+    <header class="header">
+      <div class="logo__container">
+        <a href="#home">
+          <img
+            src="images/logo.png"
+            alt="moyo-logo"
+            class="header__logo"
+          >
+        </a>
+      </div>
+
+      <nav class="header__nav">
+        <ul class="header__list">
+          <li class="header__list__item">
+            <a
+              href="#Apple"
+              class="header__list__link is-active"
+            >Apple</a>
+          </li>
+          <li class="header__list__item">
+            <a
+              href="#Samsung"
+              class="header__list__link"
+            >Samsung</a>
+          </li>
+          <li class="header__list__item">
```

Comment:
```suggestion
          <li class="header__list__item">
            <a
              href="#Apple"
              class="header__list__link is-active"
            >
              Apple
            </a>
          </li>
          
          <li class="header__list__item">
            <a
              href="#Samsung"
              class="header__list__link"
            >
              Samsung
            </a>
          </li>
          
          <li class="header__list__item">
```
fix code style all over the page